CIP-0098/README.md

```diff
@@ -0,0 +1,474 @@
+
+- Information about the current state of the blockchain
+- Details from the token's mint transaction
+- Details from previously minted tokens
+
+## Motivation: why is this CIP necessary?
+
+The four following problems describe the motive for creating this standard.
+
+### Problem 1: Storage limit
+
+Cardano is very well suited for on-chain generative tokens. Compared to other blockchains, it has a low L1 storage cost per kB, but the maximum transaction size of 16 kB is more limited than other chains.
+
+How can we create generative tokens with larger on-chain codebases without hurting the blockchain?
+
+### Problem 2: Inefficient use of block space
+
+Some existing on-chain projects on Cardano make inefficient use of block space by repeatedly storing the same monolithic blob accompanied by a few unique parameters. The result is thousands of copies of the same code, close to or at the total capacity of the 16 kB transaction limit.
+
+Without imposing more restrictions on creators, how do we drastically reduce the on-chain storage footprint of generative token collections?
+
+### Problem 3: Diversity of tools
+
+While JavaScript has become the dominant language for on-chain art, it's only a fraction of other languages and tools available to artists. Many established artists use tools that are not based on web technologies and sometimes adapt or limit their workflow to make their work run in a web browser.
+
+What is the best interface for token viewers to support a diverse set of tools and languages?
+
+### Problem 4: Archival qualities
+
+Storing all dependencies for a generative token on the blockchain is not always convenient, sensible, or even viable. Examples are p5.js, three.js, Processing, and Blender, to name a few. There is currently no way to describe token dependencies, so third parties can render or reproduce digital artifacts.
+
+What's the best way to describe external dependencies and maximize the archival qualities of digital artifacts?
+
+## Specification
+
+A single DAT is distributed over at least two tokens, represented by three possible token types: _scene_, _renderer_ and optionally, _dependency_.
+
+Typically, _scene_ tokens don't hold any code. They carry a reference to the _renderer_ token, arguments for invocation of the _renderer_ and token-specific properties. _Scene_ tokens are therefore relatively small (less than 3kB). All _scene_ tokens in a collection reference one, or at most a few, _renderer_ tokens.
+
+The code to render all _scene_ tokens is stored in a _renderer_ token, which in turn can reference _dependency_ tokens containing shared libraries or assets. _Dependency_ tokens are considered extensions to the _renderer_ and can not be referenced directly by _scene_ tokens.
+
+**Note**: The DAT Metadata Standard builds on the existing [CIP-0025](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0025) standard.
+
+## **1**. Scene
```

Comment:
```suggestion
### **1**. Scene
```
Just to match CIP template.

Reply:
Right, I copied it straight from our docs, but this makes more sense indeed.